### DISCUSSION TITLE
Ability to view Documents from cloud storage like mega.nz, dropbox, google drive

### DISCUSSION THREAD

**Discussion by Giorgos-Hellene** [YetiForceCompany/YetiForceCRM#14038] [Feature improvement]
Hi company !!
i suggest an ability to viewing Documents from cloud storage like mega.nz, dropbox, google drive. It will be very convinient because there are people that use these cloud disk. 
With this feature you will be able to attach files on your email either cloud storage or from crm

**Comment by apcloic** [comment]
Also, considering Nextcloud integration.
However, I think that such an extension require a lot of work but unfortunately, no crowdfunding has worked so far.

**Reply by mariuszkrzaczkowski** [reply]
A lot of time, first to build the logic and then to add it to the system.
So far there are no customer to cover the cost of doing this.
